terraform/modules/google-cloud/apps/relay/main.tf

```diff
@@ -239,6 +241,9 @@ resource "google_compute_instance_template" "application" {
   }
 
   metadata = {
+    # RX and TX queue count should be half of queue_count above.
+    startup-script = "#!/bin/sh\nethtool -L eth0 rx ${google_compute_instance_template.application[each.key].network_interface[0].queue_count / 2} tx ${google_compute_instance_template.application[each.key].network_interface[0].queue_count / 2}"
```

Comment:
Are these variables evaluated by terraform? Maybe it is easier to manage this via terraform variables and do the math there?

Reply:
Yeah good call. 👍